docs/user/sponsors.rst

```diff
@@ -15,39 +15,11 @@ Current sponsors
   and make Read the Docs a better service for scientific projects.
 * You? (Email us at hello@readthedocs.org for more info)
 
-Past sponsors
```

Comment:
I think it's worth leaving this here -- these folks were a large part of our success, and it definitely feels worth highlighting them.